css/main.css

```diff
@@ -0,0 +1,964 @@
+/*=== RESET TO INTUITIVE BOX LAYOUT ===*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+/*=== ICONS ===*/
+@font-face {
+  font-family: 'avengers_creative_icons';
+  src:url('/fonts/avengers_creative_icons.eot?nvpozf');
+  src:url('/fonts/avengers_creative_icons.eot?#iefixnvpozf') format('embedded-opentype'),
+    url('/fonts/avengers_creative_icons.ttf?nvpozf') format('truetype'),
+    url('/fonts/avengers_creative_icons.woff?nvpozf') format('woff'),
+    url('/fonts/avengers_creative_icons.svg?nvpozf#avengers_creative_icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'avengers_creative_icons';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+
+  /*=== BETTER FONT RENDERING ===*/
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+.icon-big {
+  font-size: 3em;
+  background-color: white;
+  color: black;
+  width: 100%;
+  border-radius: 50%;
+  padding: 1.25rem 1.25rem;;
```

Comment:
You have a double semi-colon here. Also, since you're applying the same size to all four directions, you only need to write the value once: `padding: 1.25rem;`